src/item-inspector/files-by-type/ia-files-by-type.ts

```diff
@@ -0,0 +1,100 @@
+import { MetadataResponse, File } from '@internetarchive/search-service';
+import { html, customElement, LitElement, property } from 'lit-element';
+
+interface CollapsedFileInt extends File {
+  [menuId: string]: any;
```

Comment:
Is this an arbitrary json object? If so there's a nicer modern syntax for this is `Record<string, any>`